src/cljs-lib/src/calva/pprint/printer.cljs

```diff
@@ -44,15 +44,17 @@
 
 
 (defn pretty-print-js [s {:keys [maxLength, maxDepth, map-commas?] :as all-opts}]
+  (println "all-opts" all-opts)
```

Comment:
@PEZ Did you mean to leave this `println`?

Reply:
No, that's a mistake.